Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.Droid/Contoso.Forms.Demo.Droid.csproj

```diff
@@ -169,6 +174,7 @@
     <Error Condition="!Exists('../../../packages/Xamarin.Forms.2.3.3.180/build/portable-win+net45+wp80+win81+wpa81+MonoAndroid10+Xamarin.iOS10+xamarinmac20/Xamarin.Forms.targets')" Text="$([System.String]::Format('$(ErrorText)', '../../../packages/Xamarin.Forms.2.3.3.180/build/portable-win+net45+wp80+win81+wpa81+MonoAndroid10+Xamarin.iOS10+xamarinmac20/Xamarin.Forms.targets'))" />
```

Comment:
Don't know when this line was added, but it causes the error where it can't find Xamarin.Forms.*.180 - deleting this line solves the issue